java/test/src/main/java/org/ray/api/test/BaseTest.java

```diff
@@ -37,9 +37,10 @@ public void setUpBase(Method method) {
   @AfterMethod
   public void tearDownBase() {
     Ray.shutdown();
-
-    for (File file : filesToDelete) {
-      file.delete();
+    if (filesToDelete != null) {
```

Comment:
better just initialize `filesToDelete` to an empty list.

Reply:
`BaseMultiLanguageTest` also has this issue, can you also fix that?

Reply:
Updated.

PS: `BaseMultiLanguageTest` doesn't have this issue.

Reply:
Sorry that I didn't notice that `BaseMultiLanguageTest` has another issue. Fixed.